src/exec.rs

```diff
@@ -7,7 +7,7 @@
 // At your choosing (See accompanying files LICENSE_APACHE_2_0.txt,
 // LICENSE_MIT.txt and LICENSE_BOOST_1_0.txt).
 
-use alloc::{sync::Arc, task::Wake};
```

Comment:
This line should be reverted back to how it was.  You can see all the errors on your machine by running `cargo test`.